src/services/AnalysisService.ts

```diff
@@ -692,6 +706,10 @@ class AnalysisService {
               };
         });
 
+    if (runFileHashing) {
```

Comment:
if you make the fallback for contentHashManifests and archiveFileHashManifests an empty array, you can simplify line 724 to
```
const hashManifests = archiveFileHashManifests.concat(contentFileHashManifests).filter((hm) => hm.fileHashes.length > 0);
```